src/actions/historyActions.js

```diff
@@ -164,17 +165,18 @@ export const fetchSmartWalletTransactionsAction = () => {
       smartWallet: { lastSyncedTransactionId },
     } = getState();
 
-    const activeAccount = getActiveAccount(accounts);
-    if (!activeAccount || !checkIfSmartWalletAccount(activeAccount)) return;
+    const smartWalletAccount = findFirstSmartAccount(accounts);
```

Comment:
There is a new `isActiveAccountSmartWalletSelector` selector (if you want to check the active account type)

Reply:
Here we need smart wallet account even if it's not active. Otherwise,  updated fetchTransactionsHistoryAction(true) ([L484](https://github.com/pillarwallet/pillarwallet/pull/1999/files/d6999f0931d69637f410d244b7f734e858d7b338#diff-7ea48032f831c00d653dddebf080d034R484)) would not work as expected.

Reply:
Right, this is what I was suspecting :D gut, pls ignore my suggestion above then :)